omod/src/main/resources/config.xml

```diff
@@ -172,6 +172,12 @@
     <!-- /Internationalization -->
 
     <!-- Global Properties -->
+    <globalProperty>
+		<property>coreapps.headCircumferenceConceptId</property>
+		<defaultValue>5314</defaultValue>
```

Comment:
Shouldn't we use a uuid?

Reply:
@dkayiwa , i havent got that.. like i havent understood how a uuid comes into play here.

Reply:
ok , i undertand what you meant. you wanted us to use concept uuid instead of concept id. because a UUID is usualy consistent in CIEL. But that would mean Andrew Kanter Would have to first add this concept to CIEL and i get the standard UUID to set as default value